client/modules/datafiles/src/publications/PublicationSearchToolbar/PublicationSearchToolbar.tsx

```diff
@@ -47,51 +93,57 @@ export const PublicationSearchToolbar: React.FC = () => {
           <Input
             placeholder="Author, Title, Keyword, Description, Natural Hazard Event, or Project ID"
             style={{ flex: 1 }}
+            onFocus={() => setShowTips(true)}
+            autoComplete="off"
           />
         </Form.Item>
       </Form>
-      <div>
-        <label htmlFor="nh-type-select" style={{ margin: 0 }}>
-          Natural Hazard Type
-        </label>
-        &nbsp;
-        <Select
-          style={{ width: '150px' }}
-          virtual={false}
-          allowClear
-          id="nh-type-select"
-          placeholder="All Types"
-          options={[
-            { label: 'All Types', value: null },
-            ...dropdownOptions.nhTypeOptions,
-          ]}
-          popupMatchSelectWidth={false}
-          value={searchParams.get('nh-type')}
-          onChange={(v) => setSearchParam('nh-type', v)}
-        />
-      </div>
-
-      <div>
-        <label htmlFor="publication-year-select" style={{ margin: 0 }}>
-          Year Published
-        </label>
-        &nbsp;
-        <Select
-          style={{ width: '150px' }}
-          id="publication-year-select"
-          options={[{ label: 'All Years', value: null }, ...yearOptions]}
-          allowClear
-          placeholder="All Years"
-          popupMatchSelectWidth={false}
-          virtual={false}
-          value={searchParams.get('pub-year')}
-          onChange={(v) => setSearchParam('pub-year', v)}
-        />
-      </div>
 
-      <Button type="link" onClick={() => setSearchParams(undefined)}>
-        Clear Filters
-      </Button>
+      {showTips && (
+        <Collapse
+          bordered={false}
+          defaultActiveKey={['1', '2']}
+          style={{ width: '100%' }}
+        >
+          <Panel header="Search Tips" key="1">
+            <p>
+              <strong>"exact phrase"</strong> - Use quotes to search for exact
+              phrases.
+            </p>
+            <p>
+              <strong>word1 OR word2</strong> - Find results containing either
+              word.
+            </p>
+            <p>
+              <strong>word1 AND word2, word3 word4</strong> - Use AND, commas,
+              and spaces to find results containing each word.
+            </p>
+          </Panel>
+          <Panel header="Recent Searches" key="2">
```

Comment:
This `<Panel/>` component has a `showArrow` prop that we should set to false to match the UI mockup, or else the two panels should be combined. The headers should also be bolded.